### PR TITLE
ENH: note required columns in spreadsheet

### PR DESCRIPTION
**Pull request by justcalamari**

### CODE REVIEW

**Comment by justcalamari** [comment]
What other values can we have for transport and disposition?

**Comment by CJ-Wright** [comment]
Train, fedex, airplane

Leave at BNL, dispose at BNL

File: sampledb/sample_schema.json [hunk 1]
```diff
@@ -12,14 +12,19 @@
...
+            "default" : "Returned to Home Institution",
+            "enum" : ["Returned to Home Institution",
+                      "Leave at BNL",
+                      "Dispose at BNL"]},
         "transport": {"type": "string",
             "default" : "Automobile",
-            "enum" : ["Automobile"]},
+            "enum" : ["Automobile",
+                      "Train",
+                      "FedEx",
```
Comment (CJ-Wright) [inline sampledb/sample_schema.json]: Maybe just delivery service? That way we don't need to enum every postage/package service.

Comment (justcalamari) [reply sampledb/sample_schema.json]: Sure. Does the SAF list multiple shipping services?

Comment (CJ-Wright) [reply sampledb/sample_schema.json]: I'm not certain, @dragonyanglong?

File: sampledb/sample_schema.json [hunk 2]
```diff
@@ -15,14 +15,13 @@
...
+                      "Disposed as Waste at User Facility",
+                      "Consumed/Used"]},
         "transport": {"type": "string",
             "default" : "Automobile",
             "enum" : ["Automobile",
-                      "Train",
-                      "FedEx",
-                      "Airplane"]},
+                      "Ship by Common Carrier",
+                      "Resident Material"]},
```
Comment (CJ-Wright) [inline sampledb/sample_schema.json]: :+1:

**Comment by justcalamari** [comment]
@CJ-Wright Do we need descriptions for any other fields?

**Comment by CJ-Wright** [comment]
I don't think so. LGTM

**Comment by CJ-Wright** [comment]
Is this ready for merge?

**Comment by justcalamari** [comment]
Merged by commit https://github.com/diffpy/sampledb/commit/9334eb41502f63b32fe8fc3c0bd7c8eb35017f0d

**Pull request merged**